executor/services/agent_service.py

```
@@ -183,31 +180,35 @@ def execute_task(
+                await ClaudeCodeAgent.close_client(str(task_id))
+                logger.info(
+                    f"[{_format_task_log(task_id, MISSING_SUBTASK_ID)}] Closed Claude client"
+                )
             elif agent_name == "Agno":
-                await AgnoAgent.close_client(task_id)
-                logger.info(f"[{_format_task_log(task_id, -1)}] Closed Agno client")
+                await AgnoAgent.close_client(str(task_id))
+                logger.info(
+                    f"[{_format_task_log(task_id, MISSING_SUBTASK_ID)}] Closed Agno client"
+                )
             return TaskStatus.SUCCESS, None
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)